lib/players/human.rb

```diff
@@ -0,0 +1,11 @@
+module Players
+  class Human < Player  #inherits from Player
+
+    def move(board)
+      puts "Player '#{self.token}', please enter a valid move" #asks the user for input and returns it
+      your_move = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```

Reply:
Hi J, thank you for looking over my project. I really appreciate the feedback. I saved the recommended changes you suggested and pushed them to Github. I want to caution that my code won't work completely as we discussed in Slack. Please let me know if there's anything I should change.